### ISSUE TITLE
Allow to include images in titles

### ISSUE THREAD

**Issue by olivierdalang** [naokazuterada/MarkdownTOC#43]
Hi !

When writing technical documentations with markdown, I often have an image in my titles (the icon of the tool the section describes). I'd love to be able to display that image in the TOC.

For now this 

```
## Reference

### ![Load data icon](images/icon_import.png) Load data
```

becomes this in the TOC

```
- Reference
    - !Load data icon Load data
```

Thanks !

**Comment by naokazuterada** [comment]
Thanks, I will consider your suggestion!

**Comment by naokazuterada** [comment]
This problem fixed in [2.6.0](https://github.com/naokazuterada/MarkdownTOC/releases/tag/2.6.0).
